docs/UserGuide.md

```diff
@@ -122,22 +128,47 @@ Item  “potato” successfully created
 
 Command: `[item] delete ITEM_INDEX`
 
-> Description: Deletes a specified item. Returns a warning if the item does not exist. 
+> Description: Deletes a specified item. Returns a warning if the item does not exist.
 
 ---
 
-Example: 
+Example:
 
 Input
+
 ```
 delete 1
 ```
+
 Output
+
 ```
 (Item exists): Item “potato” successfully deleted!
 (Item does not exist): No item to be found at index 1. Use “list items” or “find NAME” to find the index of the item to be deleted.
 ```
 
+#### Item Update
+
+Command: `item set ITEM_NAME [ARGS]`
+
+> Description: Updates properties of an item.
+
+---
+
+Example:
+
+Input
+
+```
+item set ARGS
+```
+
+Output
+
+```
+Item successfully updated
+```
+
```

Comment:
Probably should move this above delete to match table of content.

Also the link to tag update in table of content isn't updated so it doesn't jump here.

Reply:
Will create a separate issue for this and fix in a future PR.